kernel/src/main/java/org/kframework/parser/ParserUtils.java

```diff
@@ -141,21 +141,18 @@ else if (di instanceof Require) {
                         .filter(file -> file.exists()).findFirst();
 
                 if (definitionFile.isPresent()) {
-                    File canonical = definitionFile.get().getAbsoluteFile();
-                    try {
-                        canonical = canonical.getCanonicalFile();
-                    } catch (IOException e) {}
+                    File canonical = definitionFile.get().toPath().toAbsolutePath().normalize().toFile();
```

Comment:
This seems reasonable to me, but I'd like to point this out, just in case other people think differently:
"Eliminating ".." and a preceding name from a path may result in the path that locates a different file than the original path."
https://docs.oracle.com/javase/7/docs/api/java/nio/file/Path.html#normalize()

Reply:
I did notice that. What do other people think?
I chose to eliminate `..` because it makes the reported error messages more readable.
I saw people getting confused by kompile because they were calling different files in different locations. And it happened to me too. Having clean source paths can help with that.